### PR TITLE
Add production ironic repositories

### PR DESCRIPTION
**Pull request by elfosardo**

### CODE REVIEW

**Review by joepvd — APPROVED** [review]
/lgtm

File: core-services/release-controller/_repos/ocp-4.11-rhel-8-server-ironic.repo [hunk 1]
```diff
@@ -0,0 +1,11 @@
+[rhel-8-server-ironic-rpms]
+name = rhel-8-server-ironic-rpms
+baseurl = https://mirror2.openshift.com/enterprise/reposync/4.11/rhel-8-server-ironic-rpms/
```
[the remark this answers is not in the document]
Comment (elfosardo) [reply core-services/release-controller/_repos/ocp-4.11-rhel-8-server-ironic.repo:3]: @joepvd I can access with either mirror or mirror2, do you think we should change to mirror?

File: core-services/release-controller/_repos/ocp-4.11-rhel-8-server-ironic.repo [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+enabled = 1
```
[the remark this answers is not in the document]
Comment (elfosardo) [reply core-services/release-controller/_repos/ocp-4.11-rhel-8-server-ironic.repo:4]: this is a separate repo, it's not in the default list
to enable it you need to inject in the container explicitely

**Pull request merged**